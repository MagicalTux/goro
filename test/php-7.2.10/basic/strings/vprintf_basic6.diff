     *** Testing vprintf() : basic functionality - using exponential format ***
002- 1.000000e+3
003- int(11)
004- 1.000000e+3 2.000000e+3
005- int(23)
006- 1.000000e+3 2.000000e+3 3.000000e+3
007- int(35)
002+ 1.00000000000000e+03
003+ int(20)
004+ 1.00000000000000e+03 2.00000000000000e+03
005+ int(41)
006+ 1.00000000000000e+03 2.00000000000000e+03 3.00000000000000e+03
007+ int(62)
     ===DONE===
