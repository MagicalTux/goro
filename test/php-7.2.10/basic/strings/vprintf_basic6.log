
---- EXPECTED OUTPUT
*** Testing vprintf() : basic functionality - using exponential format ***
1.000000e+3
int(11)
1.000000e+3 2.000000e+3
int(23)
1.000000e+3 2.000000e+3 3.000000e+3
int(35)
===DONE===
---- ACTUAL OUTPUT
*** Testing vprintf() : basic functionality - using exponential format ***
1.00000000000000e+03
int(20)
1.00000000000000e+03 2.00000000000000e+03
int(41)
1.00000000000000e+03 2.00000000000000e+03 3.00000000000000e+03
int(62)
===DONE===
---- FAILED
