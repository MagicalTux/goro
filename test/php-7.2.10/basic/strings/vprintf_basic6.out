*** Testing vprintf() : basic functionality - using exponential format ***
1.00000000000000e+03
int(20)
1.00000000000000e+03 2.00000000000000e+03
int(41)
1.00000000000000e+03 2.00000000000000e+03 3.00000000000000e+03
int(62)
===DONE===